db.js

```diff
@@ -186,6 +285,16 @@ var users = (function () {
         return rows[0];
       });
     },
+    insert: function (userName) {
```

Comment:
You can remove the whole users.insert route. It was already deleted because users are created through the login route. There's no need to add it back.